integration/auth/drop_user_test.go

```diff
@@ -33,10 +33,10 @@ func TestDropUserCommand(t *testing.T) {
 	ctx, db := s.Ctx, s.Collection.Database()
 
 	// TODO https://github.com/FerretDB/FerretDB-DocumentDB/issues/864
-	_ = db.RunCommand(ctx, bson.D{{"dropUser", "a_user"}})
+	_ = db.RunCommand(ctx, bson.D{{"dropUser", "drop_a_user"}})
 
 	errcmd := db.RunCommand(ctx, bson.D{ // avoid data race with and shadowing of err in parallel subtests below
-		{"createUser", "a_user"},
+		{"createUser", "drop_a_user"},
```

Comment:
Looks weird that we create a user with the name `drop_a_user`. Maybe we could create a helper similar to `testutil.DatabaseName` and use it? As a bonus, it will check that different tests use different users.

Reply:
Thanks, created a helper to avoid username being reused.